DEVELOPER_GUIDE.md

```diff
@@ -340,6 +342,15 @@ somehow. OpenSearch plugins use this configuration to include dependencies that
 Code that is on the classpath for compiling tests that are part of this project but not production code. The canonical example
 of this is `junit`.
 
+### Gradle Plugins
+
+#### Distribution Download Plugin
+
+To override the default Distribution Download URL which always downloads latest version of OpenSearch, with custom Distribution Download URL. This allows to pull artifacts from a custom location.
+This custom Distribution Download URL can be passed from external plugins as an argument.
```

Comment:
I think this is a bit clearer:

_The [Distribution Download](buildSrc/src/main/java/org/opensearch/gradle/DistributionDownloadPlugin.java) plugin downloads the latest version of OpenSearch by default, and supports overriding this behavior by setting `customDistributionUrl`._

```
./gradlew integTest -DcustomDistributionUrl="https://ci.opensearch.org/ci/dbc/bundle-build/1.2.0/1127/linux/x64/dist/opensearch-1.2.0-linux-x64.tar.gz"
```

Reply:
Got it. Thanks @dblock for your suggestion. I think it looks cleaner this way.